backports/py-grpcio/APKBUILD

```diff
@@ -11,7 +11,7 @@ depends="py-six>=1.5.2"
 makedepends="python2-dev python3-dev py-setuptools cython-dev"
 install=""
 subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
-source="https://files.pythonhosted.org/packages/ae/b6/f58654d005ef714922ec75a3b208df31b27310cdafacc1f5d952e1a3ed30/grpcio-${pkgver}.tar.gz"
+source="https://files.pythonhosted.org/packages/c8/3d/ee9ce5f0e6bd8b868849987a7dd7d356b82ba097eee40a3bbdaa295a16f4/grpcio-1.26.0.tar.gz"
```

Comment:
Download URL format used in protobuf would be better.
https://github.com/seqsense/aports-ros-experimental/pull/157/files#diff-17a105d44f3ca8587a244c15f1d7cf31R14
This one doesn't require update on version bump.

Reply:
Addressed your comment.